.github/workflows/conformance.yml

```diff
@@ -69,9 +74,11 @@ jobs:
           if [ ${{ steps.check-release.outputs.released }} == 'true' ]; then
             helm repo add antrea https://charts.antrea.io
             helm repo update
-            helm install --namespace kube-system antrea antrea/antrea --version "${{ inputs.antrea-version }}"
+            helm install --namespace kube-system antrea antrea/antrea --version "${{ inputs.antrea-version }}" \
+              --set "${{ inputs.antrea-values }}"
           else
-            kubectl apply -f build/yamls/antrea.yml
+            helm install --namespace kube-system antrea ./antrea-chart.tgz --set "image.repository=antrea/antrea-ubuntu,image.tag=latest" \
```

Comment:
in my experience, using `helm install --namespace kube-system antrea build/charts/antrea` should work. Do you actually need to create the tgz?

Reply:
I didn't know that. Thanks for the tip.